mobile/src/Student/screens/Game/PhaseTwoBasicGamePlay/index.js

```diff
@@ -112,15 +116,51 @@ const PhaseTwoBasicGamePlay = ({
     }
 
     const correctAnswer = answerChoices.find((answer) => answer.isCorrectAnswer)
-
+    const availableHints = question.instructions
     const submittedAnswerText = `Thank you for submitting!\n\nWaiting for your teacher to advance to the next section`
 
-    let cards = [
-        <Card headerTitle="Question">
-            <ScrollableQuestion question={question} />
-        </Card>,
-        <View>
-            <Card headerTitle="Answers">
+    const timerHeading =
+        [<>
```

Comment:
I might be wrong, but any reason that this and any other defined view variables here are always inside a `[]` which means they are an array? All views are enclosed inside a fragment so there's always one fragment that encloses the subviews.

Reply:
No, you are correct! I took reference from the cards const that was there previously but this doesn't make sense to use in the case above since nothing is being iterated through. Will remove and fix the code above as well. Thanks!